docs/workflows/create_sync_publish.rst

```diff
@@ -59,6 +59,31 @@ Remote GET response:
 
 .. _versioned-repo-created:
 
+Suse Linux Enterprise Server Remote
```

Comment:
Do we want to have a SLES Remote as a title when we no longer have a special remote for it?
Maybe we still do, just wondering.

`Configuration for SLES auth`?